packages/cli/README.md

```diff
@@ -151,7 +151,7 @@ Let's continue with our `minigit` example and and create the `add` and `clone` s
 
 ```ts
 import { Args, Command, Options } from "@effect/cli"
-import { Console, Option, ReadonlyArray } from "effect"
+import { Console, Option, Array as ReadonlyArray } from "effect"
```

Comment:
wouldn't it be better to not re-alias?

Reply:
@mikearnaldi Now you can take a look ;-)